daffodil-lib/src/test/scala/LineCounter.scala

```diff
@@ -30,33 +30,48 @@ object LineCounter extends App {
   /**
    * What file extensions to count.
    */
-  val fileSuffixesToInclude = List(".scala", ".java", ".tdml", ".xsd") // not .xml as some of those are data files that are giant.
+  val fileSuffixesToInclude = List(".scala", ".java", ".tdml", ".xsd", ".tdml.xml", ".xsl", ".sch", ".sbt") // not just .xml as some of those are data files that are giant.
+
+  /**
+   * What to exclude (subversion artifacts, anything generated, version control stuff, caches, settings, etc.)
+   */
+  val directoriesToExclude = List(".svn", ".git", ".settings", "lib_managed", "src_managed", "resource_managed", "bin", ".cache", "target", "lib", ".idea")
 
   /**
    * Where is this freakin' sandbox anyway....edit to point to yours.
    */
   val root = "/home/mbeckerle/dataiti/git/"
```

Comment:
This line is specific to your layout, this code will only work on your machine. This should be generalized.